README.md

```diff
@@ -182,6 +182,23 @@ docker build -f x86-openvino.Dockerfile -t "neuralet/smart-social-distancing:lat
 docker run -it -p HOST_PORT:8000 -v "$PWD/data":/repo/data -v "$PWD/config-x86-openvino.ini":/repo/config-x86-openvino.ini -v "$PWD/certificates/processor":/repo/certs  -e TZ=`./timezone.sh` neuralet/smart-social-distancing:latest-x86_64_openvino
 ```
 
+##### Run on AMD node with a connected Coral USB Accelerator with Adaptive Learning Support
+Currently Smart Social Distancing supports Adaptive Learning framework on X86 nodes with connected USB TPU. for more information about Adaptive Learning visit [this page](https://github.com/neuralet/neuralet/tree/master/applications/adaptive-learning)
```

Comment:
Is https://github.com/neuralet/neuralet/tree/master/applications/adaptive-learning the same README than the one uploaded in `adaptive-learning/README.md`, if so please link to the smart-social-distance one

Reply:
I linked to the  original code base in smart-social-distancing one since the original one is a general solution and smart-social-distancing is just a use case. Thanks.